composer.json

```diff
@@ -26,7 +29,8 @@
     "symfony/string": "^6.4",
     "prestashop/prestashop-accounts-installer": "^1.0",
     "firebase/php-jwt": "^6.3",
-    "symfony/dotenv": "^6.4"
+    "symfony/dotenv": "^6.4",
```

Comment:
@intraordinaire @jinpresta I had to update this dependency because it caused random errors due to the incompatibility with `psr/http-message": ^2` from the core:

![Capture d’écran 2025-04-30 à 13 59 29](https://github.com/user-attachments/assets/17826fef-ab08-4204-8074-15b828957e73)

The weird thing is that this error triggers very rarely, and then disappear again which is why it was not detected earlier I don't understand exactly why, maybe because it only triggers when sentry is actually used and it's not all the time 🤷‍♂️

I tried to limit the modification for compatibility to the minimum, but you'll know better than me if even this small change can have unexpected impact

Reply:
FYI I had this error when I tried to access the Dashboard, or when I tried to configure Faceted search, so maybe the bug is only present on legacy pages